scripts/image_create_fedora_atomic_dl.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash --login
+REL_VERSION=$(curl -Ls https://download.fedoraproject.org/pub/alt/atomic/stable/ |grep -Eo 'Fedora-Atomic-25-[0-9\.]*'|sort -r|head -1)
```

Comment:
I mean the "https://download.fedoraproject.org/pub/alt/atomic/stable/" bit that is common to both REL_VERSION and DOWNLOAD_URL.